crates/jstz_cli/src/sandbox/daemon.rs

```diff
@@ -537,7 +617,7 @@ fn format_sandbox_bootstrap_accounts() -> Table {
     table
 }
 
-pub async fn run_sandbox(cfg: &mut Config) -> Result<()> {
+pub async fn run_sandbox(cfg: &'static mut Config) -> Result<()> {
```

Comment:
I assume it needs to be static as the `Future` you pass to `set_jstz_node` is static and `cfg` is captured when creating the `Future` returned by `spawn_jstz_node`. 

But `cfg` isn't actually used apart from getting paths (which are owned). I suspect we could write `spawn_jstz_node` like:
```rust
fn spawn_jstz_node(cfg: &Config) -> impl Future<Output=Result<()>> + 'static {
  let log_path = sandbox_daemon_log_path(cfg);
  let kernel_log_path = cfg.sandbox_logs_dir().join("kernel.log");
  let mut log_file = OpenOptions::new()
        .create(true)
        .write(true)
        .append(true)
        .open(log_path.clone())?;
  
  jstz_node::run(SANDBOX_LOCAL_HOST_ADDR,
        SANDBOX_JSTZ_NODE_PORT,
        &format!(
            "http://{}:{}",
            SANDBOX_LOCAL_HOST_ADDR, SANDBOX_OCTEZ_SMART_ROLLUP_PORT
        ),
        &kernel_log_path,)
}